src/main/index.js

```diff
@@ -260,7 +260,7 @@ function runApp() {
 
         const url = getLinkUrl(commandLine)
         if (url) {
-          mainWindow.webContents.send(IpcChannels.OPEN_URL, url)
+          mainWindow.webContents.send(IpcChannels.OPEN_URL, { url: url, isLaunchLink: false })
```

Comment:
Please use two separate parameters instead of using a single parameter with an object, that lets you avoid the destructuring on the other end.